### ISSUE TITLE
settings/personal not reachable using Screenreader (JAWS/NVDA)

### ISSUE THREAD

**Issue by herrfeuer** [nextcloud/server#6168]
The Settings/Personal Page is not reachable for visually disabled People using Screenreader.
All Objects in Header-Left DIV are missing tags.
### Steps to reproduce
1. Use Jaws or NVDA to navigate on a NC12 Page (apps/files)
2. Try to reach Settings/Personal Page using the Top Navigation Bar

### Expected behaviour
Screenreader tells User which Field is selected by Cursor

### Actual behaviour
Screenreader tells User nothing.

**Nextcloud version:** (see Nextcloud admin page)
12

**Updated from an older Nextcloud/ownCloud or fresh install:**
fresh

**Where did you install Nextcloud from:**
git

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.

</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.5.2
  - comments: 1.2.0
  - dav: 1.3.0
  - federatedfilesharing: 1.2.0
  - federation: 1.2.0
  - files: 1.7.2
  - files_pdfviewer: 1.1.1
  - files_retention: 1.1.2
  - files_sharing: 1.4.0
  - files_texteditor: 2.4.1
  - files_trashbin: 1.2.0
  - files_versions: 1.5.0
  - files_videoplayer: 1.1.0
  - gallery: 17.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.0.0
  - nextcloud_announcements: 1.1
  - notifications: 2.0.0
  - oauth2: 1.0.5
  - password_policy: 1.2.2
  - provisioning_api: 1.2.0
  - serverinfo: 1.2.0
  - sharebymail: 1.2.0
  - systemtags: 1.2.0
  - theming: 1.3.0
  - twofactor_backupcodes: 1.1.1
  - user_saml: 1.3.2
  - workflowengine: 1.2.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - firstrunwizard
  - survey_client
  - updatenotification
  - user_external
  - user_ldap

```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
Not relevant.
```
</details>

**Are you using external storage, if yes which one:** local
**Are you using encryption:** no
**Are you using an external user-backend, if yes which one:** SAML.

### Client configuration
**Browser:**
IE11

**Operating system:**
Windows 10 

### Logs
#### Web server error log
<details>
<summary>Web server error log</summary>

```
Not relevant.
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Not relevant.
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by MorrisJobke** [comment]
I guess @pixelipo and @jancborchardt looked into this in the past.

**Comment by pixelipo** [comment]
Reader doesn't read the `<span>` element in this?
```
<li data-id="mail" class="">
	<a href="/apps/mail/" tabindex="3">
		<img src="/apps/mail/img/mail.svg" class="app-icon">
		<div class="icon-loading-small-dark" style="display:none;"></div>
	</a>
	<span>Mail</span>
</li>
```
Then we need to add `tittle` to `<a>` - that should solve the problem.


Also related to #5822 - we need to remove deprecated (I think?) `.header-appname-container menutoggle` element - is that read by screen reader, @herrfeuer ?

We should also figure out whether the Nextcloud logo should actually be a link. That is counter-intuitive, because the `<h1>` element says "Nextcloud" (screen reader) but the link is to the "Files" app.

**Comment by jancborchardt** [comment]
Yeah @herrfeuer, can you give more info on this, how to enhance or fix it? :) Would be much appreciated

**Comment by jancborchardt** [comment]
This is fixed for Nextcloud 14 with the pull requests https://github.com/nextcloud/server/pull/6700 and https://github.com/nextcloud/server/pull/9951 (and a lot more improvements on top!) :)

Thank you @herrfeuer for reporting, let us know if there are any further issues.
